src/lib/viewers/controls/color-picker/ColorPickerPalette.tsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
+import { AnnotationColor } from '../../../AnnotationModule';
 import './ColorPickerPalette.scss';
 
 export type Props = {
-    onColorSelect: (color: string) => void;
+    onColorSelect: (color: AnnotationColor) => void;
 };
 
 export default function ColorPickerPalette({ onColorSelect }: Props): JSX.Element {
-    const colors = ['#0061d5', '#26c281', '#ed3757', '#f5b31b', '#ffd700', '#4826c2'];
+    const colors = [
```

Comment:
Does this work? `const colors = Object.values(AnnotationColor);`?

Reply:
If so, does it need to be redefined per-render?

Reply:
Yes, the above suggestion works, and no, we can define it just once.